Feat/add bug report button - User-facing GitHub links

SAK5858 opened the pull request:
Issue #39 
Added unified floating help button with conditional tutorial rendering based on page context.

### Base Behavior (ALL Pages)
- Floating help button in bottom-right corner
- Always shows:
  - Report a Bug (links to GitHub issue template)
  - Request a Feature (links to GitHub issue template)

### Page-Specific Behavior (Page View Only)
- Extends base button with 3 additional tutorial options:
  - Page Walkthrough
  - Keyboard Shortcuts
  - Word Editing
- Total of 5 options on page view
- Visual divider separates Tutorials from GitHub section

### Modifications
- Uses Django template blocks for conditional rendering
- `tutorial_section` block: Child templates override to add tutorials
- `tutorial_divider` block: Child templates override to show divider
- Default: Only Bug Report + Feature Request (no tutorials)
- Page template: Overrides blocks to inject 3 tutorials

For Testing: 
- Tested locally on homepage
- Button appears above the footer
- Dropdown opens on click with both options
- Both GitHub links work correctly
- Theme color matches "Start Scanning" button

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/6479148d-6d99-47b6-9714-5cb7f9613a42" />

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/5c70f243-c284-46c3-a74e-ab338017b090" />

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/ef55dde1-f469-41a6-a941-85a612f7dfa0" />

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/c9ff6618-9e28-46ed-9228-a7aae6e025de" />

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/aebfc234-5b9d-4ca1-a9a3-e80713d119a4" />

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/6a1a6fbf-5ab2-4e0a-a7f4-3c447b6fece6" />

- Button appears on all pages via base.html template
- Users need a GitHub account to submit issues

seintun commented:
@SAK5858 

This floating button would conflict with the existing tutorial floating button on the Page.
Since we might add a tutorial to the landing page about the recently added tables.

Should we add a link to the footer that appears on every page?

SAK5858 commented:
@seintun God catch. I didn't think of it. I can move it or we can add it on the dropdown list under the same icon. What do you think?

seintun commented:
@SAK5858 Yeah, if you can reuse the existing Floating button from there and add the Report bugs there underneath, making it consistent across pages. That would be great.


> @seintun Good catch. I didn't think of it. I can move it or we can add the tutorial in the dropdown list under the same icon. What do you think?

SAK5858 commented:
> @SAK5858 Yeah, if you can reuse the existing Floating button from there and add the Report bugs there underneath, making it consistent across pages. That would be great.
> 
> > @seintun Good catch. I didn't think of it. I can move it or we can add the tutorial in the dropdown list under the same icon. What do you think?

I made the change. However, it's on all pages, we will need to create tutorial for the other pages or focus on homepage and extraction page/Page.html

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/f0449513-66f7-4958-bdfa-3d69cd5feb90" />
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/d82e2b06-65d1-4ef3-a094-95c4e1d7f77d" />

SAK5858 commented:
> @SAK5858 You can keep the report bug as default for all pages at the bottom. Then conditionally render the tutorial based on page the user is on. Tutorials should be dynamic based on the user page landing. Also, have logic to hide tutorials if none is available.

The update is done. I also updated the description and added more screenshots

seintun commented on libriscan/biblios/templates/biblios/base.html:
```diff
@@ -8,8 +8,9 @@
-  <link href="{% static 'css/output.css' %}" rel="stylesheet" type="text/css" />
+  <!-- daisyUI CSS CDN -->
+  <link href="https://cdn.jsdelivr.net/npm/daisyui@5" rel="stylesheet" type="text/css" />
+  <script src="https://cdn.jsdelivr.net/npm/@tailwindcss/browser@4"></script>
```
Please pull stage and update your branch.
Follow ReadMe to generate output.css for your newly added css.

Your PR shouldn't revert Abdell's change and put back cdn

seintun replied:
resolved

seintun commented on package.json:
Remove package.json since we are not using nodejs for Tailwind or daisyui

seintun commented on package-lock.json:
Remove package.json since we ain't using nodejs
Pribably from your precious Tailwind setup artifacts

SAK5858 replied:
@seintun Can you check again?

seintun reviewed (changes requested):
Sorry to bounce back. Please dont change input.css and remove added package.json, which might be from your initial tailwindcss setup

Your code for floating button is still good

SAK5858 commented:
> Only request is to merge stage to avoid undo of cdn. Your code lgtm

Just merged.

seintun commented:
@SAK5858 Merge into stage if ready.
